tools/phpunit/phpunit.xml

```diff
@@ -9,6 +9,10 @@
          stopOnFailure="false"
          syntaxCheck="false"
          bootstrap="./../../vendor/autoload.php">
+    <php>
+        <!-- During testing we want all modules enabled -->
```

Comment:
Both this, and the hack in _autoload_modules were because of my inability to enable the module any other way.  You had mentioned to me previously about creating a temp dir with a minimal config.php and overriding the default configuration location - but that is beyond my current ability with ssp + php.   I agree we need a better way to do it, I'm just not sure I can create it :)